src/layout/LegacyStack/LegacyStack.tsx

```diff
@@ -50,25 +50,25 @@ const justifyMap: Record<StackJustify, string> = {
     evenly: 'tw-justify-evenly',
 };
 
-const alignMap: Record<StackAlign, string> = {
+const alignMap: Record<LegacyStackAlign, string> = {
     start: 'tw-items-start',
     end: 'tw-items-end',
     center: 'tw-items-center',
     stretch: 'tw-items-stretch',
 };
 
-export const Stack = ({
+export const LegacyStack = ({
     children,
     padding,
     spacing,
     direction = 'row',
     wrap = false,
     justify = 'start',
     align = 'stretch',
-}: StackProps): ReactElement => {
+}: LegacyStackProps): ReactElement => {
     return (
         <div
-            data-test-id="stack"
+            data-test-id="legacy-stack"
```

Comment:
We should use the `fondue-` prefix here and also add `data-test-id` as a prop like we are doing on other components now.